### PR TITLE
fix qdrant cloud quickstart documentation

### PR DESCRIPTION
**Pull request by davidmyriel**
-update steps taken to spin up instance, authentication made clearer 
-overall edits and fixes in line with technical writing standards
-will add more images, still a draft

### CODE REVIEW

File: qdrant-landing/content/documentation/cloud/cloud-quick-start.md [hunk 1]
```diff
@@ -3,45 +3,73 @@ title: Quick Start in Cloud
...
+> **Note:** The API key needs to be present in the request header every time you make a request via Rest or gRPC interface.
+
+## Create a custom cluster
+
+> **Prerequisite:** Please make sure you have provided billing information before creating a custom cluster. 
+
+1. Start in the **Overview** section of the Dashboard. 
+2. Under **Set a Cluster Up** scroll down and click **Create Custom Tier**.
+3. A window will open. Enter a cluster **Name**.
+4. In the Beta version, you can only deploy to AWS. We are currentl;y developing support GCP and Azure.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/cloud/cloud-quick-start.md:43]: "currentl;y"?

File: qdrant-landing/content/documentation/cloud/cloud-quick-start.md [hunk 2]
```diff
@@ -3,45 +3,73 @@ title: Quick Start in Cloud
...
+1. Start in the **Overview** section of the Dashboard. 
+2. Under **Set a Cluster Up** scroll down and click **Create Custom Tier**.
+3. A window will open. Enter a cluster **Name**.
+4. In the Beta version, you can only deploy to AWS. We are currentl;y developing support GCP and Azure. 
+5. Choose from one of five data center regions. If you have latency concerns or other topology-related requirements, [let us know](mailto:cloud@qdrant.io).
+6. Configure RAM size for each node (2GB to 64GB). 
+> Please read [Capacity and Sizing](https://qdrant.tech/documentation/cloud/capacity/) to make the right choice. If you need more capacity per node, [let us know](mailto:cloud@qdrant.io).
+7. Choose the number of CPUs per node (0.5 core to 16 cores). The max/min number of CPUs is coupled to the chosen RAM size. 
+8. Select the number of nodes you want the cluster to be deployed on. 
+> Each node is automatically attached with a disc space offering enough space for your data if you decide to put the metadata or even the index on the disc storage.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/cloud/cloud-quick-start.md:49]: Shouldn't it be "disk space", not "disc space"?

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
Just two minor comments

**Review by fabriziobonavita — COMMENTED** [review]
Fantastic, thank you, left some comments

File: qdrant-landing/content/documentation/cloud/cloud-quick-start.md [hunk 3]
```diff
@@ -40,16 +41,16 @@ curl \
...
 3. A window will open. Enter a cluster **Name**.
-4. In the Beta version, you can only deploy to AWS. We are currentl;y developing support GCP and Azure. 
+4. In the Beta version, you can only deploy to AWS. We are currently developing support GCP and Azure. 
 5. Choose from one of five data center regions. If you have latency concerns or other topology-related requirements, [let us know](mailto:cloud@qdrant.io).
 6. Configure RAM size for each node (2GB to 64GB). 
 > Please read [Capacity and Sizing](https://qdrant.tech/documentation/cloud/capacity/) to make the right choice. If you need more capacity per node, [let us know](mailto:cloud@qdrant.io).
 7. Choose the number of CPUs per node (0.5 core to 16 cores). The max/min number of CPUs is coupled to the chosen RAM size. 
 8. Select the number of nodes you want the cluster to be deployed on. 
-> Each node is automatically attached with a disc space offering enough space for your data if you decide to put the metadata or even the index on the disc storage.
+> Each node is automatically attached with a disk space offering enough space for your data if you decide to put the metadata or even the index on the disc storage.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/cloud/cloud-quick-start.md:50]: There is also "disc storage", which should be changed to "disk storage"

**Review by kacperlukawski — COMMENTED** [review]
Just one minor comment

**Comment by davidmyriel** [comment]
should be good to go

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
Looks good!

**Review by kacperlukawski — APPROVED** [review]

**Pull request merged**